htdocs/societe/card.php

```diff
@@ -2423,8 +2423,29 @@ function init_supplier_categ() {
 		if ($object->client)
 		{
 			print '<tr><td>';
-			print $langs->trans('CustomerCode').'</td><td>';
-			print $object->code_client;
+			print $langs->trans('CustomerCode').'<span class="opacitymedium" style=" float: right" id="clipboardChar" onclick="clipBoardCopy()"><i " class="far fa-clipboard"></i></span></td>';
```

Comment:
Because this feature can work for all fields, can you replace inline code with a call to a function
showValueWithCopyAndPasteButton(value)
So move all code on the side of value (no more on the 1st column) and into second replace, just have:
print '<td id="code_client">';
print $object->code_client
print '</td>';
with 
print '<td>';
print showValueWithCopyAndPasteButton($object, 'code_client'); 
print '</td>';
And use <span id="code_client"> around the value instead of as an id of td.

So we will be able to use it anywhere, in any context (table or not).